source/tests.d

```diff
@@ -813,3 +813,8 @@ unittest // CachedResults copies
     assert(data[0][0].as!string == "ABC");
 }
 
+unittest // UTF-8
+{
+    auto db = Database(":memory:");
+    db.run("SELECT '\u2019\u2019';");
```

Comment:
I think that SELECT return value should be tested too.

Reply:
> I think that SELECT return value should be tested too.

Sure, why not. Added a commit.